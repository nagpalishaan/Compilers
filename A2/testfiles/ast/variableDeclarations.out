Program
  List
    Function
      IdDecl ID="a"
      List
        DeclareVar
          IdDecl ID="x"
          List
        DeclareVar
          IdDecl ID="y"
          List
