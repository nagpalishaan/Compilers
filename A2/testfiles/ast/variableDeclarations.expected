Program
  List
    Function
      IdDecl ID="a"
      List
        DeclareVar
          IdDecl ID="x"
        DeclareVar
          IdDecl ID="y"
